### PR TITLE
Exposed SDS sync interval as a configuration

### PR DESCRIPTION
**Pull request by shtripat**
tendrl-bug-id: Tendrl/gluster-integration#312
Signed-off-by: Shubhendu <shtripat@redhat.com>

### CODE REVIEW

**Comment by shtripat** [comment]
@Tendrl/tendrl-core plz review

File: tendrl/gluster_integration/sds_sync/__init__.py [hunk 1]
```diff
@@ -79,7 +79,9 @@ def _run(self):
                         lock.acquire(lock_ttl=60)
                 except etcd.EtcdLockExpired:
                     continue
-                gevent.sleep(10)
+                gevent.sleep(
+                    int(NS.config.data.get("sync_interval", 10))
```
Comment (r0h4n) [inline tendrl/gluster_integration/sds_sync/__init__.py]: This is not the only place that should use `sync_interval`

All the out of band object .save() with ttl should use ttl value by adding 10-15 seconds to `sync_interval`

Comment (shtripat) [inline tendrl/gluster_integration/sds_sync/__init__.py]: Should I grep for ttl and whenever there is ttl being set for the entities, change it to sync_interval + 15 ?

Comment (r0h4n) [inline tendrl/gluster_integration/sds_sync/__init__.py]: I am ok with whatever approach you take for this

Comment (shtripat) [inline tendrl/gluster_integration/sds_sync/__init__.py]: Actually at the moment, there are no ttl set for any entities in gluster-integration. Later if @GowthamShanmugam 's out of box delete PR needs ttl, he can use the logic to set the value as `sync_interval + 15`.

@GowthamShanmugam ^^^

**Pull request merged**